decidim/decidim
# Refactor tags position in show resource page

#### :tophat: What? Why?
This PR covers the tags related changes from #13328. 
Additionally, i have cleaned up a bit the codebase: 
- Removed a partial that was used in one single place ( debates )
- Removed some partials that were only displaying the comment button with or without section wrapper

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13328

#### Testing
1. Make sure the pipeline is green 
2. Apply patch, and see that all the tags are above the like / comment button in it's own section. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
Meetings: 
![image](https://github.com/user-attachments/assets/832a2144-975d-4c51-8652-504a6ea2814d)
![image](https://github.com/user-attachments/assets/2274314e-473e-495a-bac0-b9bbeec142b9)


Debates: 
![image](https://github.com/user-attachments/assets/3b4c2247-2528-4284-a3be-41c01e35dcd1)
![image](https://github.com/user-attachments/assets/14bbbdb5-0496-4e2d-9b9c-58e692478825)


Accountability: 
![image](https://github.com/user-attachments/assets/a177da5b-b259-418c-b746-1bca86b9395d)
![image](https://github.com/user-attachments/assets/d35029f4-d017-4a89-bfe0-9c2a08b523d4)

Budgets:
![image](https://github.com/user-attachments/assets/b1de2860-7cb8-499c-b016-34f0269bd45c)
![image](https://github.com/user-attachments/assets/ccbfc1d5-606d-4f51-aae6-7588a61141f4)

Proposals:
![image](https://github.com/user-attachments/assets/eb457fea-05ac-4acf-ae3b-cb3abc269d4b)


:hearts: Thank you!
